Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+*Kubernetes + CoreOS + Cilium Stack building with Ansible*
+
+IP SPACES
+---------
+
```

Comment:
This guide uses the following IP addresses and subnets, please substitute based on your requirement and availability of IP addresses.